### PR TITLE
Building and Wonder Sprite Update

### LINKED ISSUE #941: Organize the storage of media files in the source tree
**Is your feature request related to a problem? Please describe.**
Right now we have images stored in multiple places in the source tree. Some of the images are actually duplicates. This request is to organize as many of the images and other media files (e.g. icons, music, sound) that is common to a single directory structure under `/data`.

**Describe the solution you'd like**
We have common/shared media in the following directories:
- `/data`
- `/data/buildings`
- `/data/flags`
- `/data/graphics`
- `/data/graphics/buildings`
- `/data/graphics/cimpletoon`
- `/data/graphics/logo+splash`
- `/data/graphics/tech`
- `/data/graphics/wonders`
- `/data/icons/*`
- `/data/misc`
- `/data/themes/gui-qt/icons`
- `/data/wonders`

**Describe alternatives you've considered**
Consolidate all to a `/media` structure, such as:

- `/data/media/sound`
- `/data/media/buildings`
- `/data/media/flags`
- `/data/media/icons`
- `/data/media/music`
- `/data/media/techs`
- `/data/media/tilesets`
- `/data/media/other`
- `/data/media/wonders`

Should move `/data/graphics/cimpletoon` to `/data/cimpletoon` so the files are in the tileset folder where they belong.

**Additional context**
None, just trying to organize things better.

### LINKED ISSUE #1417: Get unique sprites for LTT/LTX buildings and wonders
**Is your feature request related to a problem? Please describe.**
As we get closer to hosting LT games in freeciv21, would be great if we could have unique sprites for the different buildings and wonders we support in our online games for the tier-1 tilesets Amplio2 and Hexemplio at a minimum

**Describe the solution you'd like**
LTT missing sprites (with the fallback in parenthesis)

* Merchantile Exchange (Bank)
* Amphitheater (Colosseum)
* SDI Defense II (SDI Defense)
* SDI Defense III (SDI Defense)
* Verrocchio's Workshop (Leonardo's Workshop)
* Trade Company (Marco Polo's Embassy)
* Atlantic Telegraph Company (University)
* Mausoleum of Mausolos (Oracle)
* Hal Saflieni Hypogeum (Oracle)

LTX missing sprites

* Academy (University)
* Occupation Government (Barracks I)
* Merchantile Exchange (Bank)
* Amphitheater (Colosseum)
* Workshop (Factory)
* Workshop (Port Facility)
* SDI Defense II (SDI Defense)
* SDI Defense III (SDI Defense)
* Verrocchio's Workshop (Leonardo's Workshop)
* Trade Company (Marco Polo's Embassy)
* Atlantic Telegraph Company (University)
* The Paris Agreement (University)
* Global Emission Union (University)
* Mausoleum of Mausolos (Oracle)
* Terracotta Army (Sun Tzu's War Academy)
* School of Baudhayana Sutras (Oracle)

**Describe alternatives you've considered**
What we have now works, just think we can come up with some unique sprites to enhance the interface.

**Additional context**
Came up as part of working on #941

### LINKED ISSUE #1433: New "small" building graphics look out-of-place in Trident
**Describe the bug**

Consider the screenshot below:
image
By comparing the buildings and units, the following differences can be noticed:

* Graphics originating from Amplio are less contrasted and have strong antialiasing; Trident units don't
* Trident units have a solid black outline, the new building sprites don't
* Perspective is different:
  * Amplio buildings use a complex perspective with a horizon line and two vanishing points
  * When at all, Trident graphics use an isometric perspective (hilarious isn't it) with lines following a 2:1 slope (two pixels right, one pixel up)

Moreover, the buildings are noticeably distorted compared to their amplio2 counterparts:
image

**To Reproduce**
1. Use Trident

**Expected behavior**
Stylistic choices are uniform between units and buildings.

### PR DESCRIPTION
**Pull request by jwrober**
Closes #941
Closes #1417 
Closes #1433

### CODE REVIEW

File: data/tilesets/buildings/mercantile_exchange.license [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 2019 Sketlux
```
Comment (lmoureaux) [inline data/tilesets/buildings/mercantile_exchange.license:2]: I would argue that the bank copyright should be propagated here

File: data/tilesets/buildings/unknown.license [hunk 2]
```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 2022 Battle for Wesnoth Contributors
```
Comment (lmoureaux) [inline data/tilesets/buildings/unknown.license:2]: Digging into the Wesnoth repo gives:
```
SPDX-FileCopyrightText: 2008 alinkmaze
SPDX-FileCopyrightText: 2004 Isaac Clerencia Perez
```

File: data/tilesets/buildings/unknown.license [hunk 3]
```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (lmoureaux) [inline data/tilesets/buildings/unknown.license:1]: Not sure that this is a good sprite for an unknown building, the silhouette of a building would work better.

File: data/tilesets/buildings/occupationgov.license [hunk 4]
```diff
@@ -0,0 +1,6 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 2010 Tim F. Smith <yoohootim@hotmail.com>
+SPDX-FileCopyrightText: 2010 Frederic Rodrigo <f.rodrigo@tuxfamily.org>
+SPDX-FileCopyrightText: 2010 Dusan Hukuv <dusan@mirc.cz>
+SPDX-FileCopyrightText: 2022 James Robertson <jwrober@gmail.com>
+SPDX-FileCopyrightText: 2022 Louis Moureaux <m_louis30@yahoo.com>
```
Comment (lmoureaux) [inline data/tilesets/buildings/occupationgov.license:6]: Thanks but copyright doesn't protect ideas. Maybe could be protected if I had registered a trademark, but even then I doubt it :joy:

File: data/tilesets/buildings/amphitheater.license [hunk 5]
```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 2019 Wahazar
```
Comment (lmoureaux) [inline data/tilesets/buildings/amphitheater.license:2]: This one is older for sure

File: data/tilesets/buildings/city_hall.license [hunk 6]
```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 2019 Wahazar
```
Comment (lmoureaux) [inline data/tilesets/buildings/city_hall.license:2]: Not Wahazar's, first appeared [here](https://forum.freeciv.org/f/viewtopic.php?p=2151#p2151)

File: data/tilesets/wonders/guangzhou_port_facility.license [hunk 7]
```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 1996-2021 Freeciv Contributors
```
Comment (lmoureaux) [inline data/tilesets/wonders/guangzhou_port_facility.license:2]: [first appearance in 2017 here](https://forum.freeciv.org/f/viewtopic.php?p=93762#p93762)

**Review by lmoureaux — COMMENTED** [review]
Impressive work on the `.license` files! Spotted a couple inaccuracies.

File: data/tilesets/buildings/unknown.license (hunk 3 above)
Comment (jwrober) [reply data/tilesets/buildings/unknown.license:1]: I'm going to remove it then and we can determine what we want to do for ultimate fallback at another time.

File: data/tilesets/buildings/city_hall.license (hunk 6 above)
Comment (jwrober) [reply data/tilesets/buildings/city_hall.license:2]: thanks, good catch

**Pull request merged**